externals/kyuubi-flink-sql-engine/src/main/java/org/apache/kyuubi/engine/flink/result/ResultSet.java

```diff
@@ -108,12 +108,32 @@ public String toString() {
         + '}';
   }
 
+  /**
+   * Before version 1.15, there were concurrency issues.
```

Comment:
Could you please add a JIRA ticket in the comment? e.g.
"Before FLINK-XXXX, there were concurrency issues, which affects Flink 1.14"